ui/kotlinx-coroutines-android/r8-test-rules.pro

```diff
@@ -0,0 +1,19 @@
+# Entry point for retaining MainDispatcherLoader which uses a ServiceLoader.
```

Comment:
It looks like this rules file is mostly a copy of `r8-test-rules-no-optim.pro` with added discard check. Can we somehow make it reference the former file to ease maintenance in the future when we'll be adding more optimization rules and testing for them?